api/src/test/unit/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandlerTest.java

```diff
@@ -623,6 +624,31 @@ public void getTradeAnnualTests_Registrations_DefiningOneRegistrationReturnsOneV
         createHandlerAndGetTradeAnnualTests(request);
     }
 
+    @Test()
+    public void getTradeAnnualTests_WorksAsV6WhenNoApiVersionSpecified() throws Exception {
+        MultivaluedMap<String, String> headers = new MultivaluedHashMap<>();
+        headers.put("Accept", Arrays.asList("application/json"));
+        ArgumentCaptor<String> argumentCaptor = ArgumentCaptor.forClass(String.class);
+
+        final String registrations = "REG001, REG002";
+
+        request.setAnnualTestsRegistrations(registrations);
+
+        uk.gov.dvsa.mot.vehicle.hgv.model.Vehicle vehicle = new uk.gov.dvsa.mot.vehicle.hgv.model.Vehicle();
+        vehicle.setVehicleIdentifier("REG001");
+
+        uk.gov.dvsa.mot.vehicle.hgv.model.Vehicle vehicle1 = new uk.gov.dvsa.mot.vehicle.hgv.model.Vehicle();
+        vehicle.setVehicleIdentifier("REG002");
+
+        when(requestContext.getHeaders()).thenReturn(headers);
+        when(tradeAnnualTestsReadService.getAnnualTests(any())).thenReturn(Arrays.asList(vehicle, vehicle1));
+
+        createHandlerAndGetTradeAnnualTests(request);
+
+        verify(searchVehicleResponseMapperFactory).getMapper(argumentCaptor.capture());
+        assertNull(argumentCaptor.getValue());
+    }
+
```

Comment:
Do we need a test to exercise the handler when it's called with v7 request? (That may already be happening but I can't see where)

Reply:
Ok. I added tests checking if proper mapper version was invoked.